### ISSUE TITLE
Accessibility Metatag Bugs Should Be a Priority

### ISSUE THREAD

**Issue by mgifford** [ScanGov/scangov#157]
**Describe the bug**

If you look at:
  https://scangov.org/profile/?domain=uscurrency.gov#metadata
  
The metatags scores are:
  Grade: B / Score: 82% (9 of 11 tags)

And the Open Graph metatags are rated the same as title and viewport.

Although og metatags are important to customer experience and SEO, it isn't as important as having an accessible site.

Can you change the weighting so that accessibility is better prioritized and people are more motivated to act on it. 

**Expected behavior**
ScanGov should be aligned to government priorities and accessibility definitely is on.

**Comment by lukefretwell** [comment]
@mgifford we've added starter accessibility standards and will iterate on this. Feel free to add relevant issues that would help guide us in making it more robust.

https://scangov.org/sorts/accessibility/
